tests/src/tracing/eventcounter/gh53564.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Tests here can't be SDK style tests - refer to other tests to get a valid CSPROJ